engine/src/main/java/org/terasology/engine/modes/loadProcesses/CreateWorldEntity.java

```diff
@@ -69,7 +72,12 @@ public boolean step() {
                 }
             }
         } else {
-            EntityRef worldEntity = entityManager.create();
+            EntityRef worldEntity;
+            if (gameManifest.getWorldInfoMap().size() > 1) {
+                worldEntity = entityManager.create(gameManifest);
+            } else {
+                worldEntity = entityManager.create();
+            }
```

Comment:
creation of worldEntity and creation of worldPools are different things. Handle them differently at the right places.